scripts/demos/common/orchestration.sh

```
@@ -0,0 +1,93 @@
+    die "generated $generated still references local-only processes; update PUBLIC_UPSTREAM_EXCLUDED_PROCESSES or process-compose.yaml"
+  fi
+  echo "$generated"
+}
+
+up() {
+  have process-compose || die "process-compose not found"
+  validate_up
+  local compose_file
+  compose_file="$(process_compose_file)"
+  telemetry_up
+  cd "$SCRIPT_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)